css/styles.css

```diff
@@ -409,39 +531,64 @@ main {
 		height: 15px;
 	}
 	
+	.tildeContainer {
+		width: 600px;
+		display: block;
+	}
+
+	.tildeSpacer {
+		width: 0px;
+		display: block;
+	}
+
+	.tildeSpacer div {
+		height: 25px;
+	}
+
+	.tildeBox {
+		width: 100%;
+	}
+
+	.tildeBox div {
+		height: 25px;
+		font-size: 40px;
+	}
+
 	section {
 		width: 600px;
+		padding-top: 20px;
 	}
 	
 	.portfolioContainerTitle {
-	padding: 30px 0 0 0;
+		padding: 60px 0 0 0;
 	}
+	
 	.samplePortfolioContainer {
-		width: 98%;
+		width: 96%;
 	}
 
 	.samplePortfolioSpacer {
-		width: 1%;
+		width: 2%;
 	}
 	
 	.samplePortfolioBoxHead {
-	padding: 0;
-	margin: 115px 0 0 45px;
-	float: left;
-	font-size: 21px;
-	letter-spacing: 4.3px;
-	line-height: 38px;
+		padding: 0;
+		margin: 115px 0 0 45px;
+		float: left;
+		font-size: 21px;
+		letter-spacing: 4.3px;
+		line-height: 38px;
 	}
 	
 	.samplePortfolioBoxSubheading {
-	font-family: 'Baskerville MT Std';
-	padding: 0;
-	margin: 0 115px 0 0;
-	float: right;
-	font-style: italic;
-	font-size: 22px;
-	letter-spacing: 2.1px;
-	line-height: 24px;
+		font-family: 'Baskerville MT Std';
```

Comment:
You want to include a fallback font here, eg font-family: 'Baskerville MTStd', serif;
Same for the other font declarations.